smack-core/src/main/java/org/jivesoftware/smack/util/NumberUtil.java

```diff
@@ -28,7 +28,21 @@ public static void checkIfInUInt32Range(long value) {
             throw new IllegalArgumentException("unsigned 32-bit integers can't be negative");
         }
         if (value > ((1L << 32) - 1)) {
-            throw new IllegalArgumentException("unsigned 32-bit integers can't be greater then 2^32 - 1");
+            throw new IllegalArgumentException("unsigned 32-bit integers can't be greater than 2^32 - 1");
```

Comment:
I'd really like to see those unrelated fixes - thanks for that - in an extra commit, since they are unrelated to this change.